src/store/api/userApi/signUp.ts

```diff
@@ -28,33 +28,28 @@ export const signUp = async (
       throw new Error(errorData.message || 'Falha no registro');
     }
 
-    // Tipar explicitamente o retorno da resposta como IUser
     const newUser: IUser = await response.json();
 
-    // Supondo que o backend também retorna accessToken e refreshToken
     const { accessToken, refreshToken }: TokenResponse = {
       accessToken: newUser.refreshTokens[0]?.token,
       refreshToken: newUser.refreshTokens[0]?.token,
     };
 
-    // Armazena o accessToken e o refreshToken
     await setToken(accessToken, refreshToken);
 
-    // Atualiza o estado global com os dados do usuário
     set({
       token: accessToken,
       isAuthenticated: true,
       isLoading: false,
       error: null,
     });
 
-    // Retorna o usuário criado
     return newUser;
-  } catch (error) {
+  } catch (error: any) {
     set({
       isLoading: false,
-      error: 'Erro no registro. Verifique suas informações.',
```

Comment:
Removed all the portuguese messages for now. We should review those in the future, if it is needed.